### DISCUSSION TITLE
Custom Elements JSON Generation for Storybook Web Components Project

### DISCUSSION THREAD

**Discussion by shainaBk** [storybookjs/storybook#25320] [Help]
Hi everyone,

I’m currently working on a design system project using agnostic components developed with OpenWC, which are then imported into a main Storybook Web Components project. I’ve encountered a problem with generating the custom-elements.json file for the documentation.

Has anyone here tackled a similar project involving OpenWC components integrated into Storybook? If so, I’m keen to learn from your experiences, particularly regarding the generation of custom-elements.json for documentation purposes.

Any insights or suggestions on how to resolve this issue would be greatly appreciated. Thank you!

**Comment by usrrname** [comment]
Check out @break-stuff https://github.com/break-stuff/wc-storybook-helpers and https://github.com/break-stuff/cem-tools!
